src/.vuepress/config.js

```diff
@@ -41,7 +41,7 @@ const sidebar = {
       ]
     },
     {
-      title: 'Transitions & Animation',
+      title: 'Transiciones y Animación',
```

Comment:
I think `Animación` should be plural
```suggestion
      title: 'Transiciones & Animaciones',
```